tsconfig.node.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
did you test this out in node?

Reply:
i did not - this config might not be necessary for this package since we are focusing on web support. I added it here to be consistent with our other frontend repos. this is another thing we can centralize as you alluded to here: https://github.com/viamrobotics/viam-typescript-sdk/pull/1#discussion_r1026594211